### ISSUE TITLE
I can't locate a visual of my bitcoin address

### ISSUE THREAD

**Issue by almcmordie** [bitpay/copay#4606]
Hi,

I was sent $ but cannot locate it at the Copay app.  How do I get my bitcoin address back in the app to get the $.

**Comment by matiu** [comment]
Could you please explain the issue on more detail? Adding a screenshot could help.

**Comment by almcmordie** [comment]
Hi,  I generated an address for one of my Copay wallets around June 14.  On July 13, someone sent some bitcoin to that address.  I hadn't opened Copay since June 14.  But yesterday, I opened Copay and my wallets were all gone.  So, I imported the wallets using the 12-word phrases.  The 34-character address that consists of upper/lower case letters and numbers wasn't among the addresses listed.  I also scanned for payments.  But nothing.  When I go to the address on the blockchain, I can see the funds, but I can't see the address or the funds at Copay.  Am I describing the problem alright?  Thanks,  Al

**Comment by matiu** [comment]
Could you tell us the address? we can check is that belongs to a Copay wallet, and when it was created.

**Comment by JDonadio** [comment]
Hi @almcmordie, I'm not sure what happened in your case but if you have the wallet backup please try to use the [recovery tool](https://bitpay.github.io/copay-recovery/) to send your funds to another wallet. As Matiu says if you can provide us the address and some screenshot would be great.

**Comment by almcmordie** [comment]
The address is 1CBvdbpHjQfBLZTPrhKqm2bFaeU3q3uwGp

**Comment by matiu** [comment]
That address belongs to a Copay wallet. It was created on Jul 15, and the corresponding walletId ends with 'be87eb6f6c93'.  It is the FIRST address of the wallet, so no scanning should be needed to access the funds.

It must be that the wallet you are recovering is not the one that own that address. Check in Copay: Settings > Advanced > Wallet Info for the walletId. Check if  it ends with `be87eb6f6c93`.

**Comment by almcmordie** [comment]
Thanks Matiu.  Everything you say is correct.  I created the address on July 15.  Unfortunately, none of my wallets end in be87eb6f6c93.  I currently have 3 wallets in my Copay, but only 2 of those wallets existed on July 15.  The first address in my wallet (that I can see) ends in bd7981c345cf.  The 2nd wallet in my Copay that existed on July 15 has an ID ending in 9a49ff4e2b7e.  Those are the only 2 wallets that existed on July 15 that I have been able to recover (Is there a 3rd wallet I am missing?) Is there any other way to recover the wallet/funds?
